templates/amazon-2018.03/cgroup.conf.erb

```diff
@@ -0,0 +1,5 @@
+###
```

Comment:
can we be more generic in the directory name, e.g. amazon-2018?

Reply:
Yes, and for future reference, using only the major version will work for both template and cookbook file resources